document_page_work_instruction/readme/CREDITS.md

```diff
@@ -0,0 +1,7 @@
+The development of this module has been financially supported by:
+
+- Camptocamp
+
```

Comment:
```suggestion
```

Reply:
@Kimkhoi3010 can you address this point please?